src/extensions.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "name": "subgraph",
+    "description": "This Scaffold-ETH 2 extension helps you build and test subgraphs locally for your contracts. It also enables interaction with the front-end and facilitates easy deployment to Subgraph Studio.",
+    "repository": "https://github.com/scaffold-eth/create-eth-extensions",
+    "branch": "subgraph",
+    "installCommand": "npx create-eth@latest -e subgraph",
+    "builder": "0x1A2d838c4bbd1e73d162d0777d142c1d783Cb831",
+    "coBuilders": []
```

Comment:
I think we can remove these fields because anyways they are not being shown on the `/extensions` UI and it also decreases our overhead of adding this every time we add an extension here.

Reply:
Yeah..I think we can remove: `installCommand` (easy to infer from other fields), `builder`, or `coBuilders`

Reply:
> Yeah..I think we can remove: `installCommand`  (easy to infer from other fields)

I think `name` could be different from the name in `installCommand`. Is it mandatory to `branch` to be the same as in the `installCommand`? Both variants are good to me, just asking to know the rules

External example where all 3 are different (branch is master):
<img width="527" alt="image" src="https://github.com/user-attachments/assets/30c14814-2b96-4be4-9c65-b14e69e55063">

Reply:
also, we need `installCommand` for example because we will have hardcoded version of `create-eth` in the extensions. Or we will need a new field for `create-eth` version

Reply:
Left installCommand as optional.

Reply:
yes, let's add it